### PR TITLE
Graviton hackathon page

### LINKED ISSUE #805: Update gitcoin hackathon page for Graviton hackathon
Update gitcoin hackathon page for Graviton hackathon

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Add page for Graviton hackathon similar to Gitcoin hackathon.

## Changes

Page added. Content to be updated based on workshops planned.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

Closes #805

### CODE REVIEW

**Comment by dg-concordium** [comment]
@limemloh @Bargsteen Now that the schedule for the workshops and hackathon is set, this topic needs to be completed before the workshops begin.

**Comment by dg-concordium** [comment]
@Bargsteen @limemloh @smh1001 The Workshop sessions section needs to be updated ASAP since the hackathon is scheduled to begin on the 14th. I got all of the other details from the LinkedIn post about the hackathon.

**Review by Bargsteen — APPROVED** [review]
LGTM

File: source/mainnet/net/guides/graviton-hackathon.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+- `Building Concordium Smart Contracts <https://www.youtube.com/watch?v=9po12_IavKU>`_.
+
+- `Building A Front-end Integration <https://www.youtube.com/watch?v=ux00M5XLHZk>`_.
+
+Quick guide
+===========
+
+- Install Rust
+- Install ``cargo concordium``
```
Comment (limemloh) [inline source/mainnet/net/guides/graviton-hackathon.rst:44]:
```suggestion
- Install ``cargo concordium`` or the `Concordium smart contract VSCode extension <https://marketplace.visualstudio.com/items?itemName=Concordium.concordium-smart-contracts>`_
```

**Review by limemloh — APPROVED** [review]
Looks good

**Pull request merged**